jupyter_book/book_template/content/guide/05_advanced.md

```diff
@@ -32,6 +32,17 @@ In both cases, the cells *will* remain in the notebooks themselves, they
 simply won't show up in the site's HTML, so links that point to a JupyterHub/
 Binder/etc will still work as expected.
 
+## Enable Google Analytics
+
+If you have a Google Account, you can use Google Analytics to collect some information on the traffic to your Jupyter Book. With this tool, you can find out how many people are using your site, where they come from and how they access it, wether they are using the Desktop or the mobile version etc. 
```

Comment:
this is exactly where we'd need the docs :-) 

Could you break the lines so that they're roughly PEP8 happy?

Reply:
also done